tests/boardfarm_plugins/README.md

```diff
@@ -0,0 +1,113 @@
+### Boardfarm plugin dir structure:
```

Comment:
I realize we don't have it in any other markdown file, but we really should have a copyright header:

```suggestion
<!--
SPDX-License-Identifier: BSD-2-Clause-Patent
Copyright (c) 2020 the prplMesh contributors
This code is subject to the terms of the BSD+Patent license.
See LICENSE file for more details.
-->

### Boardfarm plugin dir structure:
```

Note that this would be the first time we use that "the prplMesh contributors" approach. If you feel uncomfortable with it, you can also just use your own name.

Reply:
Got it.
"the prplMesh contributors" is OK.